cpp/CMakeLists.txt

```diff
@@ -58,39 +58,31 @@ set(GUNROCK_GENCODE_SM72 "OFF")
 set(GUNROCK_GENCODE_SM75 "OFF")
 set(GUNROCK_GENCODE_SM80 "OFF")
 
-# Check for aarch64 vs workstation architectures
-if(CMAKE_SYSTEM_PROCESSOR MATCHES "aarch64")
-  message(STATUS "CMAKE Detected aarch64 CPU architecture, selecting appropriate gencodes")
-  # This is being build for Linux4Tegra or SBSA ARM64 CUDA
-  set(GPU_ARCHS "62") # Default minimum CUDA GenCode - not supported by gunrock
-  if(CUDA_VERSION_MAJOR GREATER_EQUAL 9)
-    set(GPU_ARCHS "${GPU_ARCHS};72")
-    set(GUNROCK_GENCODE_SM72 "ON")
```

Comment:
Where are the `GUNROCK_GENCODE_SM*` variables being set to `"ON"` for the relevant architectures? The Gunrock build needs these to be built properly, otherwise Gunrock arch build options all default to `"OFF"`.

Reply:
This was fixed in commit https://github.com/rapidsai/cugraph/pull/1267/commits/03573ecbd231395d47d26ac6c02cebe79f08227b